### PR TITLE
[AAA Dashboard] Added export to excel file for list views

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord 

I added export to the excel for the list view in the AAA Dashboard.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/aaa/models.py [hunk 1]
```diff
@@ -1193,6 +1195,35 @@ class AggregationInformation(models.Model):
     aggregation_window_end = models.DateTimeField()
 
 
+class AaaFile(models.Model):
```
Comment (emord) [inline custom/aaa/models.py]: What does this class add over calling `get_blob_db().put()` directly? It doesn't appear that we're trying to cache the downloads in anyway so I think directly calling it without addinga model is a better choice

File: custom/aaa/tasks.py [hunk 2]
```diff
@@ -104,3 +108,62 @@ def update_agg_awc_table(domain, month):
...
+            ('highRiskPregnancy', 'High Risk Pregnancy'),
+            ('noOfAncCheckUps', 'No. Of ANC Check-Ups'),
+        )
+        data = PregnantWomanQueryHelper.list(domain, selected_date, location_filters, sort_column)
+
+    if beneficiary_type == 'eligible_couple':
+        month_end = selected_date + relativedelta(months=1) - relativedelta(days=1)
+        data = EligibleCoupleQueryHelper.update_list(data, month_end)
+    elif beneficiary_type == 'pregnant_women':
+        data = PregnantWomanQueryHelper.update_list(data)
```
Comment (emord) [inline custom/aaa/tasks.py]: these could be pushed into the previous if/else statements

**Comment by lwyszomi** [comment]
Hi @emord I updated the PR, Thanks for your feedback

**Comment by lwyszomi** [comment]
@emord I added possibility to export the details reports.

File: custom/aaa/const.py [hunk 3]
```diff
@@ -3,6 +3,8 @@
 
 from django.utils.translation import ugettext_noop as _
 
+EXPIRED = 60 * 60 * 24 * 7
```
Comment (emord) [inline custom/aaa/const.py]: `BLOB_EXPIRATION_TIME` (also I think that 24 hours is fine as a defautl)

**Review by emord — APPROVED** [review]
Why is using `!important` required here? Is it possible to remove its usage for this dashboard?

**Comment by lwyszomi** [comment]
Hi @emord I updated the PR, but most of the ```!important``` we have to leave them because we need them in the print version of the report. Without them, colours are not reflected in this version of the report.

**Pull request merged**